packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -643,6 +654,55 @@ export function getValidatorApi({
       }
     };
 
+  const produceBlockV3: ServerApi<routes.validator.Api>["produceBlockV3"] = async function produceBlockV3(
```

Comment:
this is the real code segment introduced here, old `produceBlockV3` is renamed to `produceEngineOrBuilderBlock` and this one now further undertakes the task of returning blinded response for local is requested by the validator via query param